msal/tests/Test.MSAL.NET.UIAutomation/MSALMobileTestHelper.cs

```diff
@@ -119,6 +150,14 @@ private void PrepareForAuthentication(ITestController controller)
             SetUiBehavior(controller, uiBehavior);
         }
 
+        private void SetB2CInputData(ITestController controller)
+        {
+            controller.Tap(CoreUiTestConstants.SettingsPageID);
+
+            // Toggle B2C
+            controller.Tap(CoreUiTestConstants.B2CEntryID);
```

Comment:
How do you it is not already toggled?

Reply:
good idea. thanks.

---
In reply to: [232943273](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r232943273) [](ancestors = 232943273)